lib/src/impl_ffi/impl_ffi.digest.dart

```diff
@@ -62,39 +59,100 @@ abstract class _Hash implements Hash {
       return out.copy(size);
     });
   }
+
+  String get hmacJwkAlg;
+
+  String get rsaOaepJwkAlg;
+
+  String get rsaPssJwkAlg;
+
+  String get rsassaPkcs1V15JwkAlg;
```

Comment:
These need documentation comments. We're not overriding something that already documented.